### ISSUE TITLE
Bug: `no goal for reports/doid.subclass.direct-in-mondo-only.tsv`

### ISSUE THREAD

**Issue by joeflack4** [monarch-initiative/mondo-ingest#398]
## Overview
Nico is running `build-mondo-ingest`, but it errored out on `sync` ([original comment](https://github.com/monarch-initiative/mondo-ingest/pull/397#pullrequestreview-1842298506)).

## Investigation
I don't know why this is happening.

(1) There is a goal
It's entry number 3 in this multi-output goal: `$(REPORTDIR)/%.subclass.added.robot.tsv $(REPORTDIR)/%.subclass.confirmed.robot.tsv $(REPORTDIR)/%.subclass.direct-in-mondo-only.tsv $(REPORTDIR)/%.subclass.added-obsolete.robot.tsv: tmp/mondo-ingest.db tmp/mondo.db tmp/mondo.sssom.tsv`

(2) `make reports/doid.subclass.direct-in-mondo-only.tsv` works for me
<details><summary>Details</summary>
<p>

```
sh run.sh make reports/doid.subclass.direct-in-mondo-only.tsv
Running  with -Xmx20G of memory for ROBOT and Java-based pipeline steps.
...
RUST_BACKTRACE=full semsql make tmp/mondo.db -P config/prefixes.csv
make[1]: Entering directory '/work/src/ontology'
robot \
...
```

(3) Even just running `make sync` works for me

</p>
</details> 

## Sub-tasks
- [ ] Open a PR to reinstate the sync pipeline: try a fresh checkout of the repo (different folder) and see if can replicate the problem
- [ ] When done: resolve [this comment](https://github.com/monarch-initiative/mondo-ingest/pull/397#discussion_r1465494952)

**Comment by joeflack4** [comment]
@matentzn Apologies for assigning this to you as well but I can't replicate.

Maybe it has something to do with how it's being run? This happened during `build-mondo-ingest`?

**Comment by matentzn** [comment]
If you delete the sync folder first (this is my state) - can you try and replicate?

**Comment by joeflack4** [comment]
I'll try that tomorrow!

**Comment by matentzn** [comment]
This has been fixed by #403
